priv/graphql/schemas/admin/muc.gql

```diff
@@ -4,26 +4,37 @@ Allow admin to manage Multi-User Chat rooms.
 type MUCAdminMutation @protected{
```

Comment:
It is not necessary because each field in type is marked as protected. But if an unauthorized request tries to execute MUCAdmin queries then the permissions analysis could be stopped after a type check without fields checking.

I would keep this object `protected` directive because it has no negative consequences and even makes unauthorized request permissions checking faster/easier.